packages/admin/admin/src/form/FinalFormSelect.tsx

```diff
@@ -9,8 +9,8 @@ import { type AsyncOptionsProps } from "../hooks/useAsyncOptionsProps";
 import { LinearLoadingContainer, MenuItemDisabledOverrideOpacity } from "./FinalFormSelect.sc";
 
 export interface FinalFormSelectProps<T> {
-    noOptionsLabel?: ReactNode;
-    errorLabel?: ReactNode;
+    noOptionsText?: ReactNode;
```

Comment:
The rename is okay for me, but we still need a changeset

Reply:
added changeset and marked it as minor.